torch/_tensor_docs.py

```diff
@@ -3433,6 +3448,37 @@ def callable(a, b) -> number
     tensor([[2.0000, 2.0000, 3.2300, 2.0000],
             [2.0000, 2.0000, 2.0000, 3.2300]])
 
+    >>> # Testing the case where index.size(d) > src.size(d)
+    >>> src = torch.arange(1, 13).reshape((2, 2, 3))
+    >>> src
+    tensor([[[ 1,  2,  3],
+             [ 4,  5,  6]],
+
+            [[ 7,  8,  9],
+             [10, 11, 12]]])
+    >>> gen = torch.Generator()
+    >>> gen.manual_seed(123456)
+    >>> index = torch.empty((2, 3, 4), dtype = torch.long).random_(0, 3, generator = gen)
```

Comment:
Fair enough! I used the generator there because I wanted the index to be reasonably big (24 elements in that case) and I thought it'd look clunkier to write all 24 elements out by hand.